main/lsp/lsp_helpers.cc

```diff
@@ -104,6 +104,20 @@ unique_ptr<Range> loc2Range(const core::GlobalState &gs, core::Loc loc) {
     return make_unique<Range>(move(start), move(end));
 }
 
+unique_ptr<core::Loc> range2Loc(const core::GlobalState &gs, const Range &range, core::FileRef file) {
+    ENFORCE(range.start->line >= 0);
```

Comment:
At some point, we may want to do more input validation here. `range2Loc` is run on untrusted input from the client, and I think `pos2Offset` can crash if the data is garbage.

Reply:
This doesn't need to be a `unique_ptr`; `Loc` is a lightweight class Sorbet passes around by value.

Reply:
Good point. To be fair, I don't think `ENFORCE` runs in release mode, right? So these wouldn't even catch a bad client input. Do you think making this more robust should block here?

Reply:
Correct that `ENFORCE` is debug-only. This is a non-blocking comment for now. I'm learning via fuzzing that we have other non-robust functions like this one.